frontend/src/error/GenericErrorPage.tsx

```diff
@@ -75,7 +75,7 @@ const ButtonWrapper = styled.div`
   width: 100%;
 `;
 
-export const ErrorPage = ({ statusCode }: { statusCode: number }) => {
+export const GenericErrorPage = ({ message }: { message: string }) => {
```

Comment:
Ooh I find typing function components to be cleaner with the `React.FC\<PropsType\>`, since it prevents both having inline types where we are destructuring props and having to specify a return type for the component. 

Nothing too wrong with this though! I would however like to see a return type for the component(`JSX.Element` I think). I find the return type for a component to be useful in cases where I do dumb stuff and either forget to return JSX or return something else lmao.